src/course-unit/course-sequence/sequence-navigation/SequenceNavigationTabs.jsx

```diff
@@ -1,23 +1,48 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
-import { Link } from 'react-router-dom';
 import { Button } from '@edx/paragon';
 import { Plus as PlusIcon } from '@edx/paragon/icons';
 import { useIntl } from '@edx/frontend-platform/i18n';
+import { useNavigate } from 'react-router-dom';
 
-import { useIndexOfLastVisibleChild } from '../hooks';
+import { addNewSequenceNavigationUnit } from '../../data/thunk';
+import { addNewUnitId, changeTitleEditFormOpen, updateQueryPendingStatus } from '../../data/slice';
+import { getCourseId, getNewUnitId, getSequenceId } from '../../data/selectors';
 import messages from '../messages';
+import { useIndexOfLastVisibleChild } from '../hooks';
 import SequenceNavigationDropdown from './SequenceNavigationDropdown';
 import UnitButton from './UnitButton';
 
 const SequenceNavigationTabs = ({ unitIds, unitId }) => {
   const intl = useIntl();
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const sequenceId = useSelector(getSequenceId);
+  const newUnitId = useSelector(getNewUnitId);
+  const courseId = useSelector(getCourseId);
```

Comment:
[question] I think we need to use a common way to get course IDs (sequenceId, courseId, etc.)
I like your approach. And probably we should do a small refactoring for that, don't we?

Reply:
Is it related to one line instead of 3 `{ sequenceId, newUnitId, courseId } ...`?
If so I don't think it makes difference so much.

Reply:
@ihor-romaniuk In the current code base, we actively use these selectors, without directly accessing the desired state element